### DISCUSSION TITLE
establishing IKE_SA failed, peer not responding

### DISCUSSION THREAD

**Discussion by mohamedhafizelhaj** [strongswan/strongswan#1429] [Q&A]
Hello

I want to establish a site-to-site VPN connection with a peer, I have the public IP of the peer which they sometimes called it the encryption domain (what is this?), and the ike & ipsec parameters.

I completed the configuration and tried to 'ipsec start' but keep getting 'establishing IKE_SA failed, peer not responding '.

I have used ike-scan tool to confirm there is an ipsec running on that IP, but I get '0 returned handshake; 0 returned notify' which I think it means that there is no ipsec running on that IP but I am not sure. So I am asking what is the problem, is it on my side or the other side (it is hard to reach the other side to investigate more)

this is a snapshot of syslog

```
sending packet: from MYIP[500] to PEERIP[500] (1232 bytes)
charon: 07[IKE] retransmit 1 of request with message ID 0
charon: 07[NET] sending packet: from MYIP[500] to PEERIP[500] (1232 bytes)
charon: 09[IKE] retransmit 2 of request with message ID 0
charon: 09[NET] sending packet: from MYIP[500] to PEERIP[500] (1232 bytes)
charon: 10[IKE] retransmit 3 of request with message ID 0
charon: 10[NET] sending packet: from MYIP[500] to PEERIP[500] (1232 bytes)
charon: 11[IKE] retransmit 4 of request with message ID 0
charon: 11[NET] sending packet: from MYIP[500] to PEERIP[500] (1232 bytes)
charon: 14[IKE] retransmit 5 of request with message ID 0
charon: 14[NET] sending packet: from MYIP[500] to PEERIP[500] (1232 bytes)
charon: 06[IKE] giving up after 5 retransmits
charon: 06[IKE] establishing IKE_SA failed, peer not responding
```

Thanks.

**Comment by mohamedhafizelhaj** [comment]
any help ?

**Comment by misch42** [comment]
Any logs?

22.11.2022 10:52:30 Mohamed Hafiz ***@***.***>:

**Reply by mohamedhafizelhaj** [reply]
I have already attached syslog

**Reply by mohamedhafizelhaj** [reply]
@misch42 what kind of logs you need to be able to help?

**Comment by misch42** [comment]
ok. the logs show that your gateway sends out the IKE packets. 

1) Can you check with tcpdump that the packets are definitely leave your network?

2) Can you or your partner verify with tcpdump that these packets are received by the partner network?

3) What is in the logs of your partner?

4) Does your partner send out an answer? Do you recieve the answers? (tcpdump)

**Comment by mohamedhafizelhaj** [comment]
@misch42  Thanks for your reply. Using tcp dump I can verify that the IKE packets are leaving my network.

I will reach out to the other side to check what you mentioned.

Meanwhile I want to confirm if this is correct or not

`I have used ike-scan tool to confirm there is an ipsec running on that IP, but I get '0 returned handshake; 0 returned notify' which I think it means that there is no ipsec running on that IP but I am not sure.`

I forgot to mention that when I use nmap to scan for partner udp port 500 and 4500 I get open|filtered for both of them, I believe one of them must respond with open if there is an ipsec service running there (correct me if I am wrong).

Thank you again.

**Reply by mohamedhafizelhaj** [reply]
> @misch42 Thanks for your reply. Using tcp dump I can verify that the IKE packets are leaving my network.
> 
> I will reach out to the other side to check what you mentioned.
> 
> Meanwhile I want to confirm if this is correct or not
> 
> `I have used ike-scan tool to confirm there is an ipsec running on that IP, but I get '0 returned handshake; 0 returned notify' which I think it means that there is no ipsec running on that IP but I am not sure.`
> 
> I forgot to mention that when I use nmap to scan for partner udp port 500 and 4500 I get open|filtered for both of them, I believe one of them must respond with open if there is an ipsec service running there (correct me if I am wrong).
> 
> Thank you again.

Any help regarding this ?
